dawn/test/unit-test/dawn/CodeGen/CMakeLists.txt

```diff
@@ -11,8 +11,16 @@
 ##  See LICENSE.txt for details.
 ##
 ##===------------------------------------------------------------------------------------------===##
+include(GoogleTest)
 
 file(COPY input DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
 file(COPY reference DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-add_subdirectory(Cuda)
-add_subdirectory(Naive)
+
```

Comment:
the folder structure should have remained that way (see mirroring the src dir)

Reply:
Good catch!